src/mig/agent/agent.go

```diff
@@ -163,8 +163,8 @@ func runModuleDirectly(mode string, args []byte, pretty bool) (out string) {
 		return fmt.Sprintf(`{"errors": ["module '%s' is not available"]}`, mode)
 	}
 	// instanciate and call module
-	modRunner := modules.Available[mode].Runner()
-	out = modRunner.(modules.Moduler).Run(os.Stdin)
+	run := modules.Available[mode].NewRun()
```

Comment:
Yeah, the name "Moduler" seemed appropriate for values representing a module, where it used to name values representing a run of a module.